### DISCUSSION TITLE
Unable to start server to run a basic hello programme

### DISCUSSION THREAD

**Discussion by sg36sagargupta26** [actix/actix-web#2991] [Q&A] [answered]
I am unable to run actix,
My main.rs
![image](https://user-images.githubusercontent.com/19642445/223130440-4c88a70e-1d05-4c27-bee2-826361618d77.png)
My cargo.toml
![image](https://user-images.githubusercontent.com/19642445/223130898-f938a23a-1014-46a1-939d-7288200f9f2b.png)

I am getting this when running cargo run
![image](https://user-images.githubusercontent.com/19642445/223131298-4afdb98d-1c3e-4df0-86de-8a0bd19b3ebd.png)

**Comment by robjtede** [comment, marked as answer]
Run `cargo clean && rm -rf ~/.cargo/registry`, seems your registry directory got corrupted somehow.

**Reply by sg36sagargupta26** [reply]
Thanks a lot :)
